### PR TITLE
Added new entries - Zyte, Devtron, Openreplay, Stepzen, Dyrectorio, Docker

### PR DESCRIPTION
**Pull request by Ruchika30**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -604,6 +658,19 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Requirements**:
+  - The first 20 participants to get a PR accepted on those projects will win Zyte Swag.
+  - Fill this [form](https://share.hsforms.com/18bF2UnWKRmKx8AgRX-1AAg2lm14) to apply for zyte swag after PR acceptance.
+- **How to sign up**:
+  - Register offically at [hacktoberfest](https://hacktoberfest.com/participation/)
+- **Issues**:
+  - Check the list of open sourec repos by Zyte [here](https://www.zyte.com/blog/hacktoberfest-2022/)
+- **Notes**:
+  - You only need to fill [this]() form after one of your pull requests is accepted.
+  - Check out the [blog post](https://www.zyte.com/blog/hacktoberfest-2022) for more info.
```
[the remark this answers is not in the document]
Comment (Ruchika30) [reply docs/list.md:673]: DId not get this. Please help in understanding

**Comment by Ruchika30** [comment]
Hi @crweiner . I have added & updated the above changes in the PR. Looking forward to hear from you. Thanks

**Comment by vimode** [comment]
Hey @Ruchika30 thank you for contribution.👏 

The netlify bot has generated a [deploy preview](https://github.com/crweiner/hacktoberfest-swag-list/pull/668#issuecomment-1272503923) take a look at it and you will notice indentation issues for the items you submitted. If there isn't a list, you don't need to break it out on the next line. Look at the other submitted items, you will get the idea of the expected formatting.
Follow the formatting you did for Zyte, the formatting for it is properly done. 👌 

The Notes section is a note for the reader to something be aware of so that their contribution is valuable and meaningful to the company. It's not required that everything you add in there has to come from the announcement, feel free to paraphrase or post a small summary of what you gather from the provided information. 

The prize list for Devtron is already revealed. You don't need to list all of them individually but it would be nice to list the category they are like Swag Kit, Stickers,,etc.  go through all of them once you will get the idea of what to add in the Swag section. 

The details of Docker can also include about the two ways they can contribute to the Docker extension, and the link they provide for all available extensions which one can contribute to. 

In the Zyte, update the swag, you accidentally added Hacktoberfest swag. You could also add the repos one is expected to contribute to. And you forgot to add the link to their form, or you can skip over the Notes a you have added that part in the Requirements section. 

I would advice go through all details once again carefully and patiently to make a meaningful contribution. And if it too overwhelming or if you feel stuck or need help with something, feel free to ask.

**Comment by crweiner** [comment]
Hello @vimode and @Ruchika30! 

Thank you @vimode for the through review. 🤝 You're absolutely correct, and I would appreciate if @Ruchika30 could make those changes. If you'd like to submit a line-by-line review formally, let me know!

**Comment by Ruchika30** [comment]
> Hello @vimode and @Ruchika30!
> 
> Thank you @vimode for the through review. 🤝 You're absolutely correct, and I would appreciate if @Ruchika30 could make those changes. If you'd like to submit a line-by-line review formally, let me know!

Yes. I am on it @crweiner  Thanks for the review

**Comment by crweiner** [comment]
Hi again @Ruchika30! 👋 Thank you for your PR to add lots of companies to the Swag List! I've had to do several big adjustments to this PR, more than I would normally do, but I wanted to make sure that these organizations can get on the Swag List ASAP. Please be sure to follow the [Contributing guide](https://hacktoberfestswaglist.com/contributing/#how-to-format-your-contribution) more closely next time. 👍  Thank you for your time and effort in putting all of this together, and I hope you have a great Hacktoberfest! 🎃 

thumbs up

**Comment by Ruchika30** [comment]
> Hi again @Ruchika30! 👋 Thank you for your PR to add lots of companies to the Swag List! I've had to do several big adjustments to this PR, more than I would normally do, but I wanted to make sure that these organizations can get on the Swag List ASAP. Please be sure to follow the [Contributing guide](https://hacktoberfestswaglist.com/contributing/#how-to-format-your-contribution) more closely next time. 👍 Thank you for your time and effort in putting all of this together, and I hope you have a great Hacktoberfest! 🎃
> 
> thumbs up [ thumbs up ](https://camo.githubusercontent.com/301978776e4c21d49453f3a42d2fd2a1254ebb58670bd33211cf6c0d18b7398b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c34316c556a55674c4c7757727a3230772f67697068792e676966) [ ](https://camo.githubusercontent.com/301978776e4c21d49453f3a42d2fd2a1254ebb58670bd33211cf6c0d18b7398b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c34316c556a55674c4c7757727a3230772f67697068792e676966)

Thanks @crweiner .Sorry that you had to make lots of edits

**Comment by Ruchika30** [comment]
> > Hi again @Ruchika30! 👋 Thank you for your PR to add lots of companies to the Swag List! I've had to do several big adjustments to this PR, more than I would normally do, but I wanted to make sure that these organizations can get on the Swag List ASAP. Please be sure to follow the [Contributing guide](https://hacktoberfestswaglist.com/contributing/#how-to-format-your-contribution) more closely next time. 👍 Thank you for your time and effort in putting all of this together, and I hope you have a great Hacktoberfest! 🎃
> > thumbs up [ thumbs up ](https://camo.githubusercontent.com/301978776e4c21d49453f3a42d2fd2a1254ebb58670bd33211cf6c0d18b7398b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c34316c556a55674c4c7757727a3230772f67697068792e676966) [ ](https://camo.githubusercontent.com/301978776e4c21d49453f3a42d2fd2a1254ebb58670bd33211cf6c0d18b7398b/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6c34316c556a55674c4c7757727a3230772f67697068792e676966)
> 
> Thanks @crweiner .Sorry that you had to make lots of edits. I will try to do better next time

**Pull request merged**